apps/explorer/src/pages/validator/DelegatorsList.tsx

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+import { SUI_TYPE_ARG } from '@mysten/sui.js';
+import { useState } from 'react';
+
+import type { Delegators } from './ValidatorDetails';
+
+import Pagination from '~/components/pagination/Pagination';
+import { CoinFormat, useFormatCoin } from '~/hooks/useFormatCoin';
+import { AddressLink } from '~/ui/InternalLink';
+import { TableCard } from '~/ui/TableCard';
+import { TableHeader } from '~/ui/TableHeader';
+import { Text } from '~/ui/Text';
+
+const DELEGATORS_PER_PAGE = 20;
+
+export type DelegatorsListProps = {
+    delegators: Delegators;
+};
+
+function DelegationAmount({ amount }: { amount?: bigint | number }) {
```

Comment:
I think we should try to have only one component per file even if it is used only in this file, will keep files small and make us think about the component in a more 'modular' way